server/src/main/java/umm3601/user/UserController.java

```diff
@@ -33,11 +33,11 @@
  */
 public class UserController {
 
-  private static final String AGE_KEY = "age";
-  private static final String COMPANY_KEY = "company";
-  private static final String ROLE_KEY = "role";
-  private static final int REASONABLE_AGE_LIMIT = 150;
+  static final String AGE_KEY = "age";
+  static final String COMPANY_KEY = "company";
+  static final String ROLE_KEY = "role";
```

Comment:
Nic originally made these public, but that's a little more open than is needed. Package level access is enough.

Reply:
People often make "constants" public, and I think I just did that out of habit.